.github/workflows/codeql.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", "release" ]
```

Comment:
The other CI workflows only list master. Ideally we aren't pushing directly to either of these, but maybe this was appropriate to run a CI for the merge commit that is pushed by the system to these branches? Probably...